spec/index.bs

```diff
@@ -15884,7 +15882,12 @@ It guarantees that:
 
 ### Sample frequency shading ### {#sample-frequency-shading}
```

Comment:
Do we want to change the terminology here to match what some other API(s) use?

Reply:
This is a hard topic to search for, because Google REALLY wants it to be the same as Variable Rate shading, but as far as I can tell we've got:

["evaluated at sample frequency" - Direct3D](https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-to-get-sample-position)
["Per-sample shading" - OpenGL](https://www.khronos.org/opengl/wiki/Multisampling#Per-sample_shading)
["Per-Sample Function" - Metal](https://developer.apple.com/metal/Metal-Shading-Language-Specification.pdf) (Section 5.5)
["Sample Shading" - Vulkan](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#primsrast-sampleshading)

Proposing "Per-sample shading" as the header here, as that at least partially overlaps with the OpenGL, Vulkan, and Metal terminology.

Reply:
I like that and I think it's the clearest.